### PR TITLE
Adds initial holiday release freeze schedule.

### PR DESCRIPTION
**Pull request by gunsch**
Initial proposal for dates for holiday release suspensions.

Note that I'm proposing a few days around Thanksgiving, as well as an extended two-week freeze over the holidays. I'm open to pushback here but I really think we should be avoiding while most people are out, and set the expectation to others not to be expecting to release functionality over the holidays.

### CODE REVIEW

File: platform/about-vsp/policies-work-norms/deployment-policy.md [hunk 1]
```diff
@@ -1,15 +1,25 @@
...
 # Automated Deploys
 
 There are automated deploys of `vets-api` and `vets-website` Monday through Friday. The release is created from `master` at 2 pm Eastern and deployed at 3 pm Eastern. Code merged to `master` after 2 pm Eastern will not be deployed until the next business day’s release. 
 
 ## Exceptions to automated deploys
 
-If there are days or periods of time when many people will be out of the office (i.e. holidays) the automated deploys will be suspended. If people aren’t in the office to support the code going out, it’s not responsible to release. 
+If there are days or periods of time when many people will be out of the office (i.e. holidays) the automated deploys will be suspended. If people aren’t in the office to support the code going out, it’s not responsible to release.
+
+**Note**: This only applies to _automated_ deploys. User-initiated deploys (e.g. a website deploy to add an emergency banner to the homepage) will be allowed to proceed as usual.
```
Comment (batemapf) [inline platform/about-vsp/policies-work-norms/deployment-policy.md]: @gunsch added this in at the request of @jenniferlee-dsva.

File: platform/about-vsp/policies-work-norms/deployment-policy.md [hunk 2]
```diff
@@ -1,15 +1,25 @@
...
+
+### Holiday release freeze schedule
+
+Automated releases will not occur on the following days, due to holidays:
```
Comment (batemapf) [inline platform/about-vsp/policies-work-norms/deployment-policy.md]: Probably getting too pedantic, but it looks like you are using "deploys" and "releases" interchangeably. Is that intentional?

File: platform/about-vsp/policies-work-norms/deployment-policy.md (hunk 1 above)
Comment (batemapf) [reply platform/about-vsp/policies-work-norms/deployment-policy.md]: Welp, learned that the process for getting emergency banner info on the homepage changed w/in the last week 🙃See Nick's [update](https://dsva.slack.com/archives/C52CL1PKQ/p1567211994105100?thread_ts=1567176833.089400&cid=C52CL1PKQ) in Slack. Will update.

File: platform/about-vsp/policies-work-norms/deployment-policy.md (hunk 2 above)
Comment (gunsch) [reply platform/about-vsp/policies-work-norms/deployment-policy.md]: good catch --- not intentional, probably just an artifact of Google's vocabulary in my brain

File: platform/about-vsp/policies-work-norms/deployment-policy.md (hunk 1 above)
Comment (gunsch) [reply platform/about-vsp/policies-work-norms/deployment-policy.md]: is this the state you want it in now?

Comment (batemapf) [reply platform/about-vsp/policies-work-norms/deployment-policy.md]: yup!

**Review by batemapf — APPROVED** [review]
LGTM. I saw https://github.com/department-of-veterans-affairs/va.gov-team/pull/1987 is in progress, too. Will leave it to @gunsch and @accbjt to reconcile anything that needs reconciling...

**Comment by gunsch** [comment]
Hey @accbjt can you take a look at this one and see if there's anything else you want to change / if this covers the bases?

**Pull request merged**